### PR TITLE
Support custom tolerations

### PR DESCRIPTION
**Pull request by yteraoka**
Currently, this helm chart support only fixed toleration, but I want to add the custom tolerations to kuberhealty deployment.
So I added the `.Values.deployment.tolerations`.

Thanks

### CODE REVIEW

File: deploy/helm/kuberhealthy/templates/deployment.yaml [hunk 1]
```diff
@@ -99,9 +99,14 @@ spec:
...
+{{- end }}
+{{- if .Values.tolerations.master }}
       - effect: NoSchedule
         key: node-role.kubernetes.io/master
         operator: Exists
-{{- end -}}
+{{- end }}
+{{- if .Values.deployment.tolerations }}
+      {{- toYaml .Values.deployment.tolerations | nindent 6 }}
+{{- end }}
```
Comment (joshulyne) [inline deploy/helm/kuberhealthy/templates/deployment.yaml:112]: i believed this section incorrectly include the `master: false` section under `.Values.deployment.tolerations`?

**Comment by joshulyne** [comment]
@yteraoka just one comment!

**Comment by mtougeron** [comment]
@yteraoka / @joshulyne this will be a huge help for us too. Can't wait to see it merged!

**Comment by yteraoka** [comment]
@joshulyne I added blank tolerations into values.yaml as placeholder instead of comment.

**Comment by joshulyne** [comment]
> @joshulyne I added blank tolerations into values.yaml as placeholder instead of comment.

@yteraoka thanks for adding that! I believe we're trying to avoid using blank properties so that they remain out of the spec and just commenting it out instead. We set a precedent for that in this pr: https://github.com/Comcast/kuberhealthy/pull/635 and we're hoping to continue that! Sorry and thanks!!

**Comment by yteraoka** [comment]
@joshulyne I got it. I changed the blank variable to example comment. Thanks.

**Comment by joshulyne** [comment]
lgtm! thanks so much!!

**Review by joshulyne — APPROVED** [review]

**Pull request merged**